### PR TITLE
[DOCS-7533] Add calculated queries doc

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

Adds a private doc on calculated queries.

[DOCS-7533](https://datadoghq.atlassian.net/browse/DOCS-7544)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7533]: https://datadoghq.atlassian.net/browse/DOCS-7533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/explorer/calculate_queries.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Calculate Queries
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md]: Calculated Fields

File: content/en/logs/explorer/calculate_queries.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+kind: documentation
+disable_toc: false
+private: true
+---
+
+## Overview
+
+Use calculated fields when you want to apply boolean logic, perform arithmetic operations, and/or manipulate strings while querying your logs.
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md]: Calculated Fields allow you to calculate, compare, and transform your log data on-the-fly.

**Review by rtrieu — APPROVED** [review]
LGTM

File: content/en/logs/explorer/calculate_queries.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+{{< img src="logs/explorer/calculated_query.png" alt="The create a calculated field modal with a formula that concatenates the first name and last name and makes it lowercase" style="width:50%;">}}
+
+## Add a calculated field
+
+You can add a calculated field from the Log Explore or on a specific JSON attribute.
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md:16]: can remove this line

File: content/en/logs/explorer/calculate_queries.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+To add a calculated field from the Log Explorer:
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md:18]: In the Log Explorer:

File: content/en/logs/explorer/calculate_queries.md (hunk 2 above)
Comment (maycmlee) [inline content/en/logs/explorer/calculate_queries.md:10]:
```suggestion
Use calculated fields when you want to apply Boolean logic, perform arithmetic operations, and/or manipulate strings while querying your logs.
```

File: content/en/logs/explorer/calculate_queries.md (hunk 1 above)
Comment (maycmlee) [inline content/en/logs/explorer/calculate_queries.md:2]:
```suggestion
title: Calculated Fields
```

File: content/en/logs/explorer/calculate_queries.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+1. Select **Calculated field**.
+
+To add a calculated field on a specific JSON attribute:
+
+1. Navigate to [Log Explorer][1].
+1. Open a log event to the side panel.
+1. Click on a specific log attribute to open the context menu.
+1. Click **Create calculated field from <attribute>**.
+
+## Support functions and operations
```
Comment (maycmlee) [inline content/en/logs/explorer/calculate_queries.md:31]:
```suggestion
## Supported functions and operations
```

File: content/en/logs/explorer/calculate_queries.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+You can add a calculated field from the Log Explore or on a specific JSON attribute.
+
+To add a calculated field from the Log Explorer:
+
+1. Navigate to [Log Explorer][1].
+1. Click the **Add** button that is next to the search query bar.
+1. Select **Calculated field**.
+
+To add a calculated field on a specific JSON attribute:
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md:24]: Pivot directly from a specific JSON attribute:

File: content/en/logs/explorer/calculate_queries.md (hunk 5 above)
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md]: Supported

File: content/en/logs/explorer/calculate_queries.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+1. Navigate to [Log Explorer][1].
+1. Open a log event to the side panel.
+1. Click on a specific log attribute to open the context menu.
+1. Click **Create calculated field from <attribute>**.
+
+## Support functions and operations
+
+- Arithmetic functions between attributes or literals: (`+`, `-`, `*`, and `/`)
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md:33]: Arithmetic operations

File: content/en/logs/explorer/calculate_queries.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+- Comparisons operators between attributes or literals: (`>`, `>`, `=`, `<`, `<=`, `==`, and `!=`)
```
Comment (usman-dd) [inline content/en/logs/explorer/calculate_queries.md:34]: Comparisons

File: content/en/logs/explorer/calculate_queries.md (hunk 2 above)
Comment (rtrieu) [reply content/en/logs/explorer/calculate_queries.md]: I'm actually seeing it lowercased in our docs.

**Review by usman-dd — APPROVED** [review]

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**